src/main/java/org/sonarlint/intellij/ui/SonarLintIssuesPanel.java

```diff
@@ -52,6 +52,7 @@
 import java.beans.PropertyChangeListener;
 import java.util.Collection;
 import java.util.Map;
+import java.util.Set;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.Set'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)